qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/modules/kubernetes/services/conda-store/worker.tf

```diff
@@ -43,12 +27,39 @@ resource "kubernetes_service" "main" {
 }
 
 
-resource "kubernetes_deployment" "main" {
+resource "kubernetes_persistent_volume_claim" "main" {
   metadata {
-    name      = "${var.name}-conda-store"
+    name      = "${var.name}-conda-store-storage"
+    namespace = var.namespace
+  }
+  wait_until_bound = true
+  spec {
+    access_modes = ["ReadWriteOnce"]
+    resources {
+      requests = {
+        storage = var.nfs_capacity
+      }
+    }
+  }
+}
+
+
+resource "kubernetes_config_map" "conda-store-environments" {
+  metadata {
+    name      = "conda-environments"
+    namespace = var.namespace
+  }
+
+  data = var.environments
+}
+
+
+resource "kubernetes_deployment" "worker" {
+  metadata {
+    name      = "${var.name}-conda-store-worker"
     namespace = var.namespace
     labels = {
-      role = "${var.name}-conda-store"
+      role = "${var.name}-conda-store-worker"
```

Comment:
Are these roles specified somewhere? or just the labels are tagged here?

Reply:
These are just labels. No new role is being created.